shopinvader_api_address/routers/__init__.py

```diff
@@ -0,0 +1 @@
+from . import address_service
```

Comment:
```suggestion
from .address_service import address_router
```
So the router can be directly imported from `shopinvader_api_address/routers`